File: Sheet/Step 1/Step 1.2/pro1-2-1.py
def star(n):

    if 1<=n and n<=25:
        for i in range(n):
            for j in range(n):
                print('* ',end="")
            print('\n',end="")
    
n = int(input())
star(n)


# for online compiler
# def nForest(n: int) -> None:
    
#     for row in range(n):
#         for col in range(n):
#             print('* ', end="")
#         print('\n',end="")

File: Sheet/Step 1/Step 1.2/C++/pro1-2-1.cpp
#include <iostream>
using namespace std;

int main()
{
    int a = 5;

    for (int row = 1; row <= a; row++) // row to input
    {
        for (int col = 1; col <= a; col++) // col to input
        {
            cout << "* "; // print *
        }
        cout << "\n"; // for new line
    }
}
